Makefile

```diff
@@ -61,7 +61,7 @@ pkg/bindata/bindata.go: $(DATA_FILES)
 less: public/css/gogs.css
 
 public/css/gogs.css: $(LESS_FILES)
-	lessc $< $@
+	lessc $< >$@
```

Comment:
What does this extra `>` do?

Reply:
Redirect output of `lessc` from stdout to file `public/css/gogs.css`.